superdesk/io/webhooks/__init__.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013 - 2017 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+from superdesk.resource import Resource
+from superdesk.services import BaseService
+from superdesk.io.commands import update_ingest
+from flask import request, abort
+import os
+import superdesk
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class FeedinServiceWebhookAuth(object):
+
+    def authenticate(self):
+        abort(403, description='You are not authorized to access this resource')
+
+    def authorized(self, allowed_roles, resource, method):
+        try:
+            # provider's id may be used in the future
+            provider_name = request.args['provider_name']
```

Comment:
@petrjasek that's because there is a bug, I've renamed the variable but forgot to put it in the lookup, it should be used to start only the provider requested, and for now it's starting all providers, I'll fix that